dbcrossbarlib/src/clouds/gcloud/bigquery.rs

```diff
@@ -191,62 +191,12 @@ pub(crate) async fn load(
     }
 }
 
-pub(crate) async fn create_table_if_not_exists(
-    ctx: &Context,
-    dest_table: &BqTable,
-) -> Result<()> {
-    debug!(ctx.log(), "making sure table {} exists", dest_table.name(),);
-    let tmp_dir = TempDir::new("bq_mk")?;
-    let dest_schema_path = tmp_dir.path().join("schema.json");
-    let mut dest_schema_file = File::create(&dest_schema_path)?;
-    dest_table.write_json_schema(&mut dest_schema_file)?;
-    let mk_child = Command::new("bq")
-        // Use `--force` to ignore existing tables.
-        .args(&[
-            "mk",
-            "--headless",
-            "--force",
-            "--schema",
-            // --project_id actually makes this fail for some reason.
```

Comment:
nice to get rid of this stuff

Reply:
Yeah, we're going to replace it with all native Rust code sometime soon.